src/aggregation/agg_req_with_accessor.rs

```diff
@@ -339,9 +341,18 @@ fn get_missing_val(
         Key::Str(_) if column_type == ColumnType::Str => Some(u64::MAX),
         // Allow fallback to number on text fields
         Key::F64(_) if column_type == ColumnType::Str => Some(u64::MAX),
+        Key::U64(_) if column_type == ColumnType::Str => Some(u64::MAX),
```

Comment:
I don't understand what `get_missing_val` is about. 

It is clearly doing something strange. Do you remember?
Can you comment/rename to make it make sense?

Reply:
yes, there is definitely a comment missing to explain it